src/Message/AbstractRequest.php

```diff
@@ -109,6 +109,11 @@ public function getShiptocountry()
     {
         return $this->getParameter('shiptocountry');
     }
+    
```

Comment:
I was initially using `getItems` which is a method that returns something completely different than what I want.